lib/events/filesessions/fileasync_test.go

```diff
@@ -438,26 +445,26 @@ func TestUploadBackoff(t *testing.T) {
 
 	// Fix the streamer, make sure the upload succeeds
 	terminateConnectionAt.Store(0)
-	p.clock.BlockUntil(2)
+	p.clock.BlockUntilContext(ctx, 2)
```

Comment:
The Flaky Test Detector looks to maybe be hanging here?


https://github.com/gravitational/teleport/actions/runs/18951112917/job/54115715130?pr=60542#step:10:748

```
goroutine 17077 [select]:
github.com/jonboulle/clockwork.(*FakeClock).BlockUntilContext(0xc00521f4a0, {0x4597898, 0xc0004c1110}, 0x2)
	/go/pkg/mod/github.com/jonboulle/clockwork@v0.5.0/clockwork.go:244 +0xfb
github.com/gravitational/teleport/lib/events/filesessions.TestUploadBackoff(0xc001ca0fc0)
	/__w/teleport/teleport/lib/events/filesessions/fileasync_test.go:448 +0xcb8
testing.tRunner(0xc001ca0fc0, 0x42389d0)
	/opt/go/src/testing/testing.go:1934 +0x21d
created by testing.(*T).Run in goroutine 1
	/opt/go/src/testing/testing.go:1997 +0x9d3

```